fetch_depth_layer/src/depth_layer.cpp

```diff
@@ -395,4 +398,16 @@ void FetchDepthLayer::depthImageCallback(
   }
 }
 
+void FetchDepthLayer::activate()
+{
+  onInitialize();
```

Comment:
I assumed yes, since those buffers would most likely be out of date anyways, it wouldn't make sense to keep using them for clearing. The other costmap plugins also reinitialize during activation, so it seemed consistent as well.